### ISSUE TITLE
Marker to follow grid rotation

### ISSUE THREAD

**Issue by antoniolocandro** [qgis/QGIS#42960]
There are already several options for grids to follow grid rotation/orthogonalize, I was thinking if this was possible to be extended to markers, I see there is like a placeholder maybe it was already thought off and not implemented.

![image](https://user-images.githubusercontent.com/4627073/116158628-35ac4480-a6ac-11eb-93e0-27b095e2b4dc.png)

**Comment by agiudiceandrea** [comment]
@antoniolocandro are you searching for a way for grid types Cross and Markers rotation or for the frame style with Ticks?
The "Follow grid rotation" settings group visible in your second screenshot is for Ticks alignments and it will be enabled if you select a frame style with Ticks.

**Comment by antoniolocandro** [comment]
Yeah ticks are OK but I have markers instead of ticks as you can have only interior or exterior ticks but in this case the markers are along the lines of the grid, slightly different use case

**Comment by agiudiceandrea** [comment]
> I see there is like a placeholder maybe it was already thought off and not implemented.

So, the "Follow grid rotation" settings group is not a placeholder. It's only about Ticks rotation.
A similar setting should be implemented for Cross and Markers grid types.
